examples/notebooks/beam-ml/run_inference_multi_model.ipynb

```diff
@@ -520,7 +520,6 @@
         "\n",
         "  def process(self, element):\n",
         "    image_url, image = element \n",
-        "    # Update this step when this ticket is resolved: https://github.com/apache/beam/issues/21863\n",
```

Comment:
Hm, is there a better way to communicate these kinds of workarounds are temporary?

Reply:
If not, we should at least comment on the issue itself with a note to update the notebook when it is fixed

Reply:
I think this is the difference between GitHub content and Devsite content. We can definitely note this as a workaround, but we need to be careful about the phrasing. I'll take a look at the Beam issue and see what I can come up with.

Reply:
Ok, thanks!

Reply:
Replaced with:

# The following lines provide a workaround to turn off BatchElements.

In all four spots.